geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/vector/SimpleFeatureVector.scala

```diff
@@ -132,16 +135,20 @@ object SimpleFeatureVector {
   val DescriptorKey   = "descriptor"
   val OptionsKey      = "options"
 
-  case class SimpleFeatureEncoding(fids: Option[Encoding], geometry: Encoding, date: Encoding)
+  case class SimpleFeatureEncoding(fids: Option[Encoding], geometry: Encoding, date: Encoding, axisOrder: Encoding)
```

Comment:
trying to map axisOrder onto min/max doesn't make too much sense here. I think it would be easier just to make it a `longitudeFirst: boolean`. then we also don't need to worry about importing the CRS libraries just to get the AxisOrder enum

Reply:
What about instead of using a boolean using the same approach that [BinaryOutputEncoder](https://github.com/locationtech/geomesa/blob/df730a143b22b27cd25f7524aec1f89b6462d612/geomesa-utils-parent/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/bin/BinaryOutputEncoder.scala) takes which is using this [AxisOrder](https://github.com/locationtech/geomesa/blob/df730a143b22b27cd25f7524aec1f89b6462d612/geomesa-utils-parent/geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/bin/AxisOrder.scala) enum?

It would be a more consistent approach throughout the Geomesa repo, wdyt?

Reply:
It's a possibility - although I'll note that even though that AxisOrder is defined in the BinaryOutputEncoder, we haven't wired through the ability to actually change it through a query hint or anything else, see e.g. [here](https://github.com/locationtech/geomesa/blob/df730a143b22b27cd25f7524aec1f89b6462d612/geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/iterators/BinAggregatingScan.scala#L41) where it's not passed through.

The main reason I prefer a boolean is that it's simple to encode/decode to string (for passing as user opts), and doesn't require the enum to be on the classpath.

Maybe we could call the hint `flipXy` - that way, it would satisfy the immediate need but would be somewhat tangential to a more complete axis order evaluation, if we ever decided to pursue one. I think the "correct" solution here is to inspect the CRS or other hints, in the same way that geoserver does, and determine axis order based on that. but due to the complexity of that approach (and to keep back-compatibility), just having a toggle seems simplest.